hw05_parallel_execution/run.go

```diff
@@ -10,6 +11,69 @@ type Task func() error
 
 // Run starts tasks in n goroutines and stops its work when receiving m errors from tasks.
 func Run(tasks []Task, n, m int) error {
-	// Place your code here.
+	// n - количество горутин
+	// m - максимальное количество ошибок
+
+	if m <= 0 {
+		return ErrErrorsLimitExceeded
+	}
+
+	tasksChan := make(chan Task)
+	errorsChan := make(chan struct{}, m)
```

Comment:
```suggestion
	errorsChan := make(chan error, m)
```
Ожидаемо что если у нас есть errCh то мы храним там errors

Reply:
Этого больше нет